packages/train2go-bridge/test/parser.test.js

```
@@ -131,6 +131,37 @@ describe("parser", () => {
     it("returns empty array on null input", () => {
       expect(parseDailyHtml(null)).toEqual([]);
     });
+
+    it("should extract description for activity intensities other than default (regression: -medium/-low/-high)", () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)